cypress/support/elements/admin_page_elements.js

```diff
@@ -5,7 +5,9 @@ const adminPageElements = {
   NOTICES_LINK : 'a[href$=\'/admin/site_notices\']',
   TAGS_LINK: 'a[href$=\'/admin/tags\']',
   SCHOOLS_LINK : 'a[href$=\'/portal/schools\']',
-  LNK_AUTHORING: 'a[href$=\'/authoring\']'
+  LNK_AUTHORING: 'a[href$=\'/authoring\']',
```

Comment:
These changes are not used yet. But I will be using these page objects for next tests